core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -89,95 +86,96 @@
           </div>
 
           <div class="oppia-navbar-role-indicator" ng-if="$ctrl.isAdmin || $ctrl.isModerator">
-            <!--
-             "right: 4px;" is necessary here but not in moderator to prevent 'A'
-             from appearing off-center because 'A' is slightly thinner than 'M' in
-             this font.
-            -->
+          <!--
```

Comment:
Keep this indented by 2, like it was before, since it refers to the following HTML tag which is indented by 2.

Reply:
Done